src/main/resources/db/migration/V24__add_city_column.sql

```diff
@@ -0,0 +1,10 @@
+alter table dokazovi.public.doctors
+add column CITY_ID INTEGER
+    CONSTRAINT DOCTORS_CITIES_ID_FKEY
+        REFERENCES CITIES;
+
+alter table users
+    add column MODIFIED_AT timestamp not null default now() ;
+
+update doctors
+set city_id = 1;
```

Comment:
Not sure, but we can get doctor's city via institutions and set them accordingly.